tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -51,43 +51,25 @@ RUN npm clean-install --no-audit --no-fund --registry=$NPM_REGISTRY \
 {{ patch("mfe-dockerfile-post-npm-install") }}
 COPY --from={{ app["name"] }}-src /openedx/app /openedx/app
 COPY --from={{ app["name"] }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
-ENV PUBLIC_PATH='/{{ app["name"] }}/'
 EXPOSE {{ app['port'] }}
 
-COPY ./env/production /openedx/env/production
-{%- set overrides = app.get("env", {}).get("production", {}) %}
-{%- if overrides %}
-RUN echo "setting production overrides..." \
-  {%- for key, value in app.get("env", {}).get("production", {}).items() %}
-  && echo "{{ key }}='{{ value }}'" >> /openedx/env/production \
-  {%- endfor %}
-  && echo "done setting production overrides"
-{%- endif %}
+# configuration needed at build time 
+ENV APP_ID={{ app["name"] }}
+ENV PUBLIC_PATH='/{{ app["name"] }}/'
+ENV MFE_CONFIG_API_URL=/api/mfe_config/v1
```

Comment:
I have double checked for this `PUBLIC_PATH` at development mode, I think it doesn't matter, it will work fine, though it can be set aet `/` _The default_, given that i development each MFE has it's domain/port.

Reply:
This is the production part of the Dockerfile, so we can't change it here.  We'd have to set `ENV MFE_CONFIG_API_URL=<something else>` in the `(dev)` section below.  But I think I prefer your proxy solution!  That way, things are more similar between production and dev.